### ISSUE TITLE
Problem of cache dir

### ISSUE THREAD

**Issue by VincentClair** [adlarge/fixtures-documentation-bundle#31]
In FixturesDocumentationManager.php, path to var directory is hardcoded.
It leads to a bug on our system as this directory is not writable, and cache dir is set to another location outside project.

```
$this->jsonFilePath = $this->projectDir . '/var/' . self::FILE_NAME;
```

Could you change it to "%kernel.cache_dir%" or something similar (or provide a way to configure the path of the path) ?

**Comment by adlarge** [comment]
Indeed this is an issue, i'm looking at it. I prefer providing a way to override this parameter because the cache dir is not necessarily the good location.

**Comment by adlarge** [comment]
Fixed in 1.2.1
